nwc_backend/__init__.py

```diff
@@ -58,6 +58,18 @@ def ready() -> str:
 
     app.register_blueprint(frontend_api_bp, url_prefix=base_path + "api")
 
+    @app.route(f"{base_path}config.js")
```

Comment:
nit: Should this have a slash in it, so `f{base_path}/config.js`?

Reply:
No, this is correct as-is, but removed this block anyway.